### PR TITLE
Esc fix

### PR DESCRIPTION
**Pull request by eranok**
Quick esc fix. Now to go off fullscrean press SHIFT+`

### CODE REVIEW

File: scripts/main.js [hunk 1]
```diff
@@ -23,8 +23,13 @@ app.whenReady().then(() => {
...
 
-  // Prevent ESC from exiting fullscreen
-  globalShortcut.register('Esc', () => { });
+  globalShortcut.register("Shift+`", () => {
+    var fs = BrowserWindow.getAllWindows()[0].isFullScreen();
+    if(fs)
+      BrowserWindow.getAllWindows()[0].setFullScreen(false);
+    else
+      BrowserWindow.getAllWindows()[0].setFullScreen(true);
+  });
```
Comment (eranok) [inline scripts/main.js:32]: could be done in one line `BrowserWindow.getAllWindows()[0].setFullScreen(!BrowserWindow.getAllWindows()[0].isFullScreen());`
i'm not good with js :)

**Comment by hmlendea** [comment]
I'm going to try this out as soon as I can.
Thanks for the suggestion

**Comment by hmlendea** [comment]
Ok so I've tested it and it seems to work for the most part.

There is a situation where it doesn't, though.
If you exit fullscreen (shift+`) and the GFN popup about going back into the game appears, if you click on the button to go into the game, it goes back to fullscreen. However, if you do that, the next time you press ESC it will both activate whatever esc is bound to in the game, as well as exiting fullscreen again (and funnily enough this time a popup is not displayed and it actually allows you to play the game windowed, lol)

I'm suggesting to merge this as-is for now, since this is a pretty big issue that's been bugging us for a long time, and the fix is definetely good enough.
This is an edge case and nothing game-breaking, just a minor annoyance.
I'd merge this now and investigate/fix this smaller issue afterwards, instead of delaying the whole fix any longer.

What do you think?

**Comment by eranok** [comment]
I am good with it :)

File: scripts/main.js (hunk 1 above)
Comment (hmlendea) [reply scripts/main.js:32]: Personally I have a C#/.NET background where having long single-line code or method calls within method calls is usually frowned upon.

A small "refactoring" will be done later to conform with the JavaScript standards but there are some other parts of the (previous) code that need changing and I'd prefer addressing them all in one PR.

So for now I've merged the branch as it was.

**Pull request merged**